.circleci/config.yml

```diff
@@ -129,7 +129,7 @@ commands:
                 conda create -y -n habitat python=3.6
                 . activate habitat
                 conda install -q -y -c conda-forge ninja numpy pytest pytest-cov ccache hypothesis
-                pip install pytest-sugar pytest-xdist
+                pip install pytest-sugar pytest-xdist pytest-benchmark
```

Comment:
This is neat, but does it actually run on CI? I think benchmarking is disabled when parallel xdist is run so if you want it to actually print out you need to select the benchmark tests and run them without parallelism.

Reply:
You can do this easily with a pytest.mark decorator for benchmark tests.

Reply:
Thanks for bringing this up. I haven't gone as far as to hook this up to mean something in CI. It would be ideal to have some cached results to compare against for automated regression testing, but printing the results would at least provide a standard environment for manual comparison.

Reply:
@aclegg3 PyTest-Benchmark has the ability to do that, but that can wait for another PR to do that and just print out the stats similar to above.